src/data/pnat/basic.lean

```diff
@@ -327,9 +343,11 @@ theorem dvd_antisymm {m n : ℕ+} : m ∣ n → n ∣ m → m = n :=
 theorem dvd_one_iff (n : ℕ+) : n ∣ 1 ↔ n = 1 :=
  ⟨λ h, dvd_antisymm h (one_dvd n), λ h, h.symm ▸ (dvd_refl 1)⟩
 
+/-- If `(n m : ℕ+)`, then `nat.gcd n m` is also positive. -/
```

Comment:
Hmmm, maybe on such definitions we want a bit more prosaic docstrings? This one will show up when someone greps for "greatest common divisor".
```suggestion
/-- The greatest common divisor (gcd) of two positive natural numbers, viewed as positive natural number. -/
```